org.eclipse.xtext.ide.tests/src/org/eclipse/xtext/ide/tests/server/RenameIssue520Test.xtend

```diff
@@ -1,47 +0,0 @@
-/*******************************************************************************
```

Comment:
why delete the test?

Reply:
where the other expectations wrong?

Reply:
It's duplicate. Please see the comment below.

Reply:
Yes, the expectations were bogus.

Reply:
ahh yes did not see that the other test actually tested the wrong thing.